src/core/execute.c

```diff
@@ -732,21 +732,21 @@ static int ask_for_confirmation(const char *vc, const char *cmdline) {
                 return 1;
         }
 
-        r = ask_char(&c, "yns", "Execute %s? [Yes, No, Skip] ", cmdline);
+        r = ask_char(&c, "yfs", "Execute %s? [Yes, Fail, Skip] ", cmdline);
```

Comment:
it would probably be a good idea to accept "n" for compat here too (but not show it on screen)